Source/Dafny/Compilers/Compiler-Csharp.cs

```diff
@@ -1636,13 +1701,21 @@ protected class ClassWriter : IClassWriter {
         }
         cw.StaticMemberWriter.WriteLine(";");
       } else if (cw.CtorBodyWriter == null) {
```

Comment:
It seems that the two cases are necessary to distinguish between the situation in which the initialization directly follows the declaration and the situation in which the initialization happens in the constructor. I tried merging these two together but it amounts to an even larger line count because of the differences in how semicolons should be treated, etc, so I would keep the two cases separate, if possible.

Reply:
I was thinking something like:

```csharp
nding;
if (isPublic) {
  cw.InstanceMemberWriter.Write($"{publik} {virtuall} {typeName} {name} {{get; set;}}");
  ending = "";
} else {
  cw.InstanceMemberWriter.WriteLine($"{publik} {typeName} {name}");
  ending = ";";
}
if (cw.CtorBodyWriter == null) {
  if (rhs != null) {
    cw.InstanceMemberWriter.Write($" = {rhs}");
  }
} else {
  if (rhs != null) {
    cw.CtorBodyWriter.WriteLine($"this.{name} = {rhs};");
  }
}
cw.InstanceMemberWriter.WriteLine(ending);
```